README.md

```diff
@@ -2,103 +2,9 @@
 
 This helper script is provided to help you simplify the onboarding to the preview of ARC on Azure VMWare Solution.
 
-## Prerequisites
+## Arc on AVS public doc
 
-Ensure you meet all the prerequisites mentioned below:
-
-1. A jump box Virtual Machine (VM) with network access to the Azure VMware Solution vCenter. From the jump-box VM, ensure you have access to vCenter and NSX-T portals. 
-
-2. Internet access from jump box VM.
```

Comment:
https://learn.microsoft.com/en-us/azure/azure-vmware/deploy-arc-for-azure-vmware-solution?WT.mc_id=Portal-Microsoft_Azure_Resources&tabs=windows#appendices

Reply:
I don't think so we have to include these URLs, we haven't explicitly mentioned these in previous README too. These are anyways present in the public docs. LMK otherwise.

Reply:
We have agreed to use only public doc link in readme and effort of maintenance of URLs is only at public docs.